skflow/__init__.py

```diff
@@ -117,6 +117,15 @@ def predict(self, X):
             return pred
         return pred.argmax(axis=1)
 
+    def predict_proba(self, X):
+        pred = self._session.run(self._model.predictions,
```

Comment:
I'll merge yours in and will implement the suggestion from above as I'm also adding scoring with mini batches and streaming and will close this PR. Thanks!